scripts/prohibit-properties-unused-keys

```diff
@@ -10,7 +10,7 @@ bundle=frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties
 
 # Let's exclude the keys that are used by IntelliJ implicitly as action descriptions (see `<override-text>` tags in plugin.xml).
 all_expected_keys=$(
-  grep --only-matching --perl-regexp '^[^=]*(?==)' -- $bundle \
+  grep --only-matching --perl-regexp '^[^= ]*(?==)' -- $bundle \
```

Comment:
previously it failed at 
```
action.GitMachete.SyncSelectedToParentByMergeAction.warning-dialog.message.HTML=\
  <html>\
    Using merge to sync branches is discouraged, especially with stacked PRs. \
    Consider using rebase instead to keep the git history linear \
    (<a href="https://github.com/VirtusLab/git-machete/blob/master/README.md#can-i-use-git-merge-for-dealing-with-stacked-prs">see FAQ</a>). \
    Do you want to proceed?\
  </html>
```
due to `<a href...`. 
(it was not caught in single string version of the text)